backend/pushpin_config/pushpin.conf

```diff
@@ -16,7 +16,7 @@ stats_connection_ttl=120
 
 [runner]
 # services to start
-services=mongrel2,m2adapter,zurl,pushpin-proxy,pushpin-handler
+services=condure,zurl,pushpin-proxy,pushpin-handler
```

Comment:
Due to this change, we probably need to test this properly in all cases where we do use pushpin:
- tests.
- docker image.
- running locally.

Reply:
also look like `pushpin_cypress_config` miss this change.

Reply:
this is no more used `mongrel2,m2adapter` ?

I think we have specific creation log file for this in docker image.

Reply:
We seen some specific configuration here https://github.com/tracim/tracim/blob/develop/tools_docker/Debian_Uwsgi/common.sh

maybe in other files.

Reply:
not anymore, the Rust language hype happened here: https://blog.fanout.io/2020/08/11/rewriting-pushpins-connection-manager-in-rust/

Reply:
`pushpin_cypress_config` is not used anymore either I forgot to erase it when `run_dev_backend.sh` has been merged or it was re-introduced accidentally. I'll remove it.
I'll update the logs so that condure logs are exposed outside of the docker container.

Reply:
In fact in `condure` cannot be installed in stretch, so the docker image still uses `mongrel2`. I'm checking whether using `mongrel2` in `docker-compose.yml` is possible but it seems more difficult than I would have liked…

Reply:
ok.